### ISSUE TITLE
Static Declaration of Function Overloads

### ISSUE THREAD

**Issue by bdrillard** [cqframework/clinical_quality_language#416]
Would it be possible to add an XML element to perhaps [`Expression`](https://github.com/cqframework/clinical_quality_language/blob/master/Src/cql-lm/schema/elm/expression.xsd#L175) or [`OperatorExpression`](https://github.com/cqframework/clinical_quality_language/blob/master/Src/cql-lm/schema/elm/expression.xsd#L183) that would declare statically which CQL [types](https://github.com/cqframework/clinical_quality_language/blob/master/Src/cql-lm/schema/elm/types.xsd) an expression is defined against? 

Backend implementations could take a much more methodical, generative approach to stubbing out Operators for their respective overloaded Types if the definitions contained the above info. Naturally the semantics of the different implementations couldn't be defined beyond doc, but this change would go a long way to generating a full inventory of operator stubs or abstract implementations.

**Comment by brynrhodes** [comment]
@bdrillard, yes this is certainly very useful, check out the [`signatures`](https://github.com/cqframework/clinical_quality_language/pull/415) translator option, I think it already supports what you're asking for.
